Containerfile.archlinux

```diff
@@ -0,0 +1,46 @@
+FROM archlinux:latest
+
+ENV CMD_pacman_INSTALL="pacman -S --needed --noconfirm"
+
+# Install useful tools.
+RUN ${CMD_pacman_INSTALL} && pacman -S base-devel git sudo bash vim nano emacs rsync cpio grub systemd man-db
+# Install compression tools.
+RUN ${CMD_pacman_INSTALL} gzip unzip zip zstd xz-utils unrar
+# Install network tools.
+RUN ${CMD_pacman_INSTALL} curl iproute iputils netcat nmap openssh-clients openssl wget
+# Install programming languages and tools.
+RUN ${CMD_pacman_INSTALL} git git-review gcc golang java jre-openjdk | grep jre | grep jdk
```

Comment:
A few issues here:

- `git-review` is only available from the AUR. We can remove this for now.
- In Arch Linux, the `golang` package is simply called `go`.
- Remove the `| grep jre | grep jdk` at the end as they are needed and cause this `RUN` instruction to fail.